tensorflow/core/kernels/cwise_ops.h

```diff
@@ -47,7 +47,11 @@ template <typename T>
 struct scalar_asinh_op {
   EIGEN_EMPTY_STRUCT_CTOR(scalar_asinh_op)
   EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE const T operator()(const T& a) const {
+#ifdef EIGEN_HAS_CXX11_MATH
```

Comment:
This should be #if EIGEN_HAS_CXX11_MATH to be consistent with Eigen and allow the code to compile on Windows and Android platforms.

Reply:
Done